<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Add Transaction</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="intakeType">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>51</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Income</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Expense</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="date">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addTransactionButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Transaction</string>
     </property>
    </widget>
    <widget class="QComboBox" name="category">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Work</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Food &amp; Drink</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bills &amp; Utilities</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Groceries</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shopping</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Personal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount: $</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="amount">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <property name="text">
     <string>Finance Tracker</string>
    </property>
   </widget>
   <widget class="QLabel" name="balance">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="income">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="expense">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
